footprint_kicad/Connector_RJ.pretty/RJ45_BEL_SS74301-00x_Vertical.pro

```diff
@@ -0,0 +1,33 @@
+update=22/05/2015 07:44:53
```

Comment:
Not sure where this file came from @rally314 @dking32 . Did you mean for it to be here?

Reply:
Not sure where it came from, it looks like it was added around the time that I was merging master into brake_sensor, but I didn't mean to add it. Do you want me to delete that file?